### DISCUSSION TITLE
How can I install GitHub Apps across the Enterprise?

### DISCUSSION THREAD

**Discussion by pasokan-ddn** [community/community#41103] [Enterprise]
How can I install GitHub Apps across the Enterprise?

Now I install GitHub App Organization level and use it on the actions controller but what if I want to deploy action controller authentication with GitHub Apps Enterprise level

**Comment by Timmmy-nlb** [comment]
This is a enterprise problem i am also facing.
And with new orgs I still have to rework them.

**Comment by hpsin** [comment]
You can't do it via a single click, but today's launch lets you automate the installation of apps in your enterprise: https://github.blog/changelog/2025-07-01-enterprise-level-access-for-github-apps-and-installation-automation-apis/
